assets/js/components/NotFound/NotFound.test.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { fireEvent, screen } from '@testing-library/react';
+import { faker } from '@faker-js/faker';
+import { useNavigate } from 'react-router-dom';
+
+import { renderWithRouter } from '@lib/test-utils';
+
+import NotFound from '.';
+
+jest.mock('react-router-dom', () => ({
+  useNavigate: jest.fn(),
+}));
+
+describe('NotFound', () => {
+  it('should render NotFound component correctly with default parameter values', () => {
+    const navigate = jest.fn();
+    useNavigate.mockReturnValue(navigate);
```

Comment:
```suggestion

```

Reply:
Ah thanks! Not only this can go, but also we don't need  `const navigate = jest.fn();` and we can use render instead of `renderWithRouter` as NotFound page does not require `useNavigate ` mocks any more :D